podvm/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include Makefile.inc
+
+.PHONY: image binaries clean
+
+UBUNTU_IMAGE_URL  := https://cloud-images.ubuntu.com/$(UBUNTU_RELEASE)/current/$(UBUNTU_RELEASE)-server-cloudimg-$(ARCH).img
+UBUNTU_IMAGE_CHECKSUM := $(shell curl -LO  https://cloud-images.ubuntu.com/"$(UBUNTU_RELEASE)"/current/SHA256SUMS && \
+				grep "$(UBUNTU_RELEASE)"-server-cloudimg-"$(ARCH)".img SHA256SUMS | awk '{print $$1}' && \
+				rm -f SHA256SUMS)
+
+IMAGE_SUFFIX := .qcow2
+PODVM_DISTRO := ubuntu
```

Comment:
Should this be `PODVM_DISTRO ?= ubuntu`?

Reply:
yes.